aoc/src/main/scala/aoc2015/exercise02.scala

```diff
@@ -0,0 +1,34 @@
+package aoc2015
+
+import scala.io.Source
+
+object exercise02 extends App:
+
+  def calcSides(l:Int, w: Int, h: Int): List[Int] =
```

Comment:
this implementation would benefit from modelling a `case class Box(l: Int, w: Int, h: Int)` - ie. a data type - with its derived proporties as methods on that data type - you now return a list of calculations while you could return a list of `Box`es .

Reply:
in other words - try to give names to the calculations you perform - eg.:

```
case class Box(l: Int, w: Int, h: Int)
  def paperMargin: Int = ??? // some implementation in terms of l, w and h
```

 and the map transformations below vv become much more readable